### PR TITLE
Updated all example snippets to use Heracles.ts reference client

### PR DESCRIPTION
**Pull request by alien-mcl**
As in the title - I've updaded all example code snippets to `typescript` using [Heracles.ts](/HydraCG/Heracles.ts)

### CODE REVIEW

File: 1.entry-point.md [hunk 1]
```diff
@@ -9,12 +9,21 @@ So I can start managing my events.
...
+const entryPoint = await hydraClient.getApiDocumentation("http://example.com/").getEntryPoint();
+
+// Gather operations exposed by first-level hypermedia controls
+for (const operation of entryPoint.hypermedia.where(_ => _.operations.any())) {
     // do something with the *operation*
 }
+
+// Gather collections
+for (const collection of entryPoint.hypermedia.collections) {
+    const add = collection.operations.ofType("http://schema.org/addAction");
```
Comment (angelo-v) [inline 1.entry-point.md]: schould be `AddAction` with captial `A`

Comment (alien-mcl) [reply 1.entry-point.md]: Ahh - thanks!

**Comment by tpluscode** [comment]
@alien-mcl I merged #1. Let'd get this one rolling once you resolve the conflicts

**Comment by alien-mcl** [comment]
OK - I've resolved conflicts. Unless there are no other comments, this one is good to go.

**Comment by angelo-v** [comment]
Ok, lets merge this

**Pull request merged**